test/katex-spec.js

```diff
@@ -185,37 +185,53 @@ describe("A subscript and superscript parser", function() {
     it("should produce supsubs for superscript", function() {
         const parse = getParsed("x^2")[0];
 
-        expect(parse.type).toBe("supsub");
-        expect(parse.value.base).toBeDefined();
-        expect(parse.value.sup).toBeDefined();
-        expect(parse.value.sub).toBeUndefined();
+        expect(parse).toMatchObject({
```

Comment:
If parse contains extra keys, they're ignored which is what we want in this test.